src/code42cli/cmds/users.py

```diff
@@ -41,13 +41,20 @@ def users(state):
     "--user-id", help="The unique identifier of the user to be modified.", required=True
 )
 
+org_id_option = click.option(
+    "--org-id",
+    help="The identifier for the organization to which the user will be moved.",
+    required=True,
+    type=int,
+)
+
 
 def role_name_option(help):
     return click.option("--role-name", help=help)
 
 
-def username_option(help):
-    return click.option("--username", help=help)
+def username_option(help, required=False):
```

Comment:
options are not required by default

Reply:
Yes, but I needed to require it for the `users move` method, but I didn't want to change any of the other methods which have it as an optional option, so I added this way to pass that in from the decorator if we wanted, but left it defaulting to False. If there's a better way to do this, I'm happy to change it.

Reply:
Oh my bad,  I overlooked the wrapper